user/python-typing_extensions/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-typing_extensions"
```

Comment:
this package already exists

but even if it didn't you don't need it, it just involves replacing `typing_extensions` with `typing`, they're backports of types